ddtrace/tracer/option.go

```diff
@@ -1006,6 +1010,14 @@ func WithPartialFlushing(numSpans int) StartOption {
 	}
 }
 
+// WithStatsComputation enables client-side stats computation.
+// This can also be configured by setting DD_TRACE_STATS_COMPUTATION_ENABLED to true.
```

Comment:
Is there a public docs link we can point to about this?
If not, we can say something like:
```
// WithStatsComputation enables client-side stats computation, allowing
// the tracer to compute stats from traces. This can reduce network traffic
// to the Datadog Agent, and produce more accurate stats data.
// This can also be configured by setting DD_TRACE_STATS_COMPUTATION_ENABLED to true.
// Client-side stats is off by default.
```

Reply:
There is no public docs, I'll apply the suggestion, thanks!